cdk/lib/bigquery-acquisitions-publisher.ts

```diff
@@ -66,7 +72,34 @@ export class BigqueryAcquisitionsPublisher extends GuStack {
         region: ["eu-west-1"],
       },
       eventBus: eventBus,
-      targets: [new SqsQueue(queue)],
+      targets: [new SqsQueue(acquisitionsQueue)],
+    });
+
+    const softOptInConsentSetterQueue =
+      aws_sqs.Queue.fromQueueArn(this, "SoftOptInConsentSetterQueue", props.softOptInConsentSetterQueueArn)
+
+    // Rule which passes events from support-workers on to soft opt-in queue
+    new Rule(this, "SoftOptInToSQSRule", {
```

Comment:
```suggestion
    new Rule(this, "SoftOptInQueueRule", {
```

Reply:
Maybe it's changed since, but I thought the plan was to put this rule in one of the support-service-lambdas CDKs, perhaps the soft opt in one? And you converted that over to CDK in order to make that possible?